pkg/cmd/admin/router/router.go

```diff
@@ -255,18 +265,28 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 			fmt.Fprintf(out, "password for stats user %s has been set to %s\n", cfg.StatsUsername, cfg.StatsPassword)
 		}
 
+		externalHostInsecure := "false"
```

Comment:
I'm following the style from 15 lines above.  Should both use `strconv.FormatBool`, or am I missing some subtlety?